controller/search_blackbox_test.go

```diff
@@ -622,6 +622,18 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
 		require.Len(s.T(), result.Data, 3) // resolved items having sprint1 are 3
 	})
 
+	s.T().Run("state=resolved AND iteration=sprint1 using EQ", func(t *testing.T) {
+		filter := fmt.Sprintf(`
+			{"$AND": [
+				{"state": {"$EQ": "%s"}},
+				{"iteration": {"$EQ": "%s"}}
+			]}`,
+			workitem.SystemStateResolved, sprint1.ID)
+		_, result := test.ShowSearchOK(s.T(), nil, nil, s.controller, &filter, nil, nil, nil, nil, &spaceIDStr)
+		require.NotEmpty(s.T(), result.Data)
+		require.Len(s.T(), result.Data, 3) // resolved items having sprint1 are 3
```

Comment:
Suggestion: You could also validate that the three wi contain the correct data values. But it isn't a blocker